### ISSUE TITLE
After upgrading to stable 1.12.13+hotfix.5 card_settings has many errors

### ISSUE THREAD

**Issue by NTMS2017** [codegrue/card_settings#82]
After upgrading to stable 1.12.13+hotfix.5 card_settings has many errors. Can you update card_settings please? 

I don't know how to build a plugin. I am using AndroidStudio and IntelliJ. If I do pull request how can I update card_settings?

I am using intl: ^0.15.8 plugin in my app. With new stable version I am getting this error.

```
Because ntms_cep depends on flutter_localizations any from sdk which depends on intl 0.16.0, intl 0.16.0 is required.

So, because ntms_cep depends on intl ^0.15.8, version solving failed.
pub get failed (1; So, because ntms_cep depends on intl ^0.15.8, version solving failed.)
```

When I use new version of plugin intl: ^0.16.0 this time I am getting card_settings error.

```
Because card_settings 1.4.2 depends on intl ^0.15.6 and no versions of card_settings match >1.4.2 <2.0.0, card_settings ^1.4.2 requires intl ^0.15.6.

So, because ntms_cep depends on both intl ^0.16.0 and card_settings ^1.4.2, version solving failed.
pub get failed (1; So, because ntms_cep depends on both intl ^0.16.0 and card_settings ^1.4.2, version solving failed.)
```

I also want to use auto_size_text: ^2.1.0 plugin. But card_settings uses auto_size_text: ^1.0.0 plugin.

It will be great to update card_setting plugin. Kind Regards

**Comment by codegrue** [comment]
Issue is related to flutter upgrading their intl dependency to 1.16.0.

**Comment by NTMS2017** [comment]
Thanks

**Comment by NTMS2017** [comment]
I try to do it my self but I am getting this error.

Because card_settings >=0.1.11 depends on intl ^0.15.6 and ntms_cep depends on intl ^0.16.0, card_settings >=0.1.11 is forbidden.

So, because ntms_cep depends on card_settings 1.4.2, version solving failed.
pub get failed (1; So, because ntms_cep depends on card_settings 1.4.2, version solving failed.)

**Comment by codegrue** [comment]
fixed in 1.5.0

**Comment by NTMS2017** [comment]
Thanks, I was reading the changeling this morning :) Kind Regards
